shared/src/models/vehicle-template.ts

```diff
@@ -45,14 +61,24 @@ export class VehicleTemplate {
         image: ImageProperties,
         patientCapacity: number,
         personnel: readonly PersonnelType[],
-        materials: readonly CanCaterFor[]
+        materials: readonly CanCaterFor[],
+        imagesPatientsLoaded?: ImageProperties[]
     ) {
         this.vehicleType = vehicleType;
         this.name = name;
         this.image = image;
         this.patientCapacity = patientCapacity;
         this.personnel = personnel;
         this.materials = materials;
+
+        if (
+            imagesPatientsLoaded !== undefined &&
+            imagesPatientsLoaded.length + 1 !== patientCapacity
+        )
+            throw new ReducerError(
```

Comment:
I'm not sure a `ReducerError` is the best choice here. @Dassderdie What do you think?